doc/playbook.rst

```diff
@@ -78,9 +78,14 @@ The bucket ``rubin:rubin-pp-users`` holds:
 * ``rubin:rubin-pp-users/unobserved/`` contains raw files that the upload script(s) can draw from to create incoming raws.
 
 ``rubin:rubin-pp`` has had notifications configured for it; these publish to a Kafka topic.
+To check what messages are sent, do port-forward:
+.. code-block:: sh
+    kubectl port-forward svc/kafdrop -n kafka 9000:9000
+
+Then messages can be viewed locally via Kafdrop at `http://127.0.0.1:9000/topic/rubin-prompt-processing`_.
```

Comment:
This would be one way to check the file arrival notifications from the bucket, not the nextVisit messages.  If I understand correctly, they are on separate kafkas (nextVisit from sasquatch's and prompt processing has its own).

I got the tip from [this slack thread ](https://lsstc.slack.com/archives/C0345PUU4N9/p1692740715825959?thread_ts=1692738210.807919&cid=C0345PUU4N9) when we were debugging some ceph setup.  I swear it was working at one point, probably from my laptop. 

However,  it seems no longer working probably because we didn't move the service to phalanx when we transitioned.  I'll check with Dan.   I'll leave out this part as it isn't really relevant to this ticket anyway.

Reply:
My mistake with mixing up the two Kafkas. Unfortunately, the instructions on the Slack thread also don't make it clear which system you're supposed to do what on...